lib/components/Button/Button.scss

```diff
@@ -0,0 +1,241 @@
+@import '../../theme/tokens/facade';
+
+@mixin isState($state, $nest: false) {
```

Comment:
We don't seem to be using the nest option anywhere

Reply:
Its more just there in case we need it in the future. Think I did use it at some point.